src/translation/translation_fr_FR.ts

```diff
@@ -3428,7 +3428,7 @@ Nous avons coupé votre canal et activé &quot;Me silencer&quot;. Veuillez d&apo
     <message>
         <location filename="../serverdlg.cpp" line="737"/>
         <source>Now a directory server</source>
```

Comment:
Yeah, this didn't get updated to remove "server".  It should have been.

Reply:
done for the translation. waiting for the english version. :-)